plugins/Marketplace/resources/qml/ManagePackagesButton.qml

```diff
@@ -38,8 +38,10 @@ Button
 
         width: UM.Theme.getSize("section_icon").width
         height: UM.Theme.getSize("section_icon").height
-
+  
```

Comment:
Please remove the trailing space

Reply:
fixed in https://github.com/Ultimaker/Cura/pull/10739/commits/246537f2f751538731de6a71ce5bdf8c0c5a47c7